kubernetes/spilo_kubernetes.yaml

```diff
@@ -132,47 +127,77 @@ spec:
 apiVersion: v1
 kind: Endpoints
 metadata:
-  name: &cluster_name zalandodemo01
+  name: *cluster_name*
   labels:
     application: spilo
-    spilo-cluster: *cluster_name
+    spilo-cluster: *cluster_name*
 subsets: []
 
+# Services
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: *cluster_name*-master
+  labels:
+    application: spilo
+    spilo-cluster: *cluster_name*
+spec:
+  type: ClusterIP
+  selector:
+    application: spilo
+    spilo-cluster: *cluster_name*
+    role: master
+  ports:
+  - name: postgresql
+    port: 5432
+    targetPort: 5432
+  - name: patroni
+    port: 8008
```

Comment:
We don't need and don't want to expose the Patroni REST API on the leader service.